docs/api-docs/catalog/v2-vs-v3.md

```diff
@@ -0,0 +1,131 @@
+<div class="HubBlock-content">
+
+> ### Note
+>
+> Product [variant options](https://developer.bigcommerce.com/api-reference/store-management/catalog/product-options/getoptionbyid) and [modifiers](https://developer.bigcommerce.com/api-reference/store-management/catalog/product-modifiers/getmodifierbyid) created using the V3 Catalog API belong to a single product instance and cannot be attached to other products in the catalog.
+
+</div>
+</div>
+</div>
+
+The following workflows demonstrate the difference between creating a product with variants using V2 and V3 Catalog APIs.
+
+**V2 API workflow to create a product with variants:**
+
+1. Create a product using the V2 Products API.
+2. Create options.
+3. Create an option set using the V2 Option Sets API.
+4. Link the product to the option set.
+5. Create adjustments, such as price adjustment, using rules.
+
+**V3 Catalog API workflow to create a product with variants:**
+
+1. [Create a product](https://developer.bigcommerce.com/api-reference/store-management/catalog/products/createproduct) with variants by sending a `POST` request to `/v3/catalog/products`. You can specify the price directly on the variant level using the `price` property of the variant object.
+
+**Product variants**
+
+In V3, every purchasable entity in the catalog is a variant including the product itself. This structure enables enhanced inventory management flows, such as using the [Variants](https://developer.bigcommerce.com/api-reference/store-management/catalog/variants/getvariants) endpoint to manage inventory levels. While it is possible to create an SKU with a subset of product options using the V2 API, V3 requires creating variants for every combination of option values. We recommend creating products using the V3 API as BigCommerce intends to move operations to the V3 API.
+
+To reduce the number of API calls made, you can include variants with a `GET` request using the `?include=variants` query parameter.
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--warning">
+<div class="HubBlock-content">
+
+<!-- theme: warning -->
+
+### V2 SKU rules will override variant pricing
+>
+> Creating SKU rules via the V2 API or CSV import will alter or override any variant price or sale price added to a product using the control panel, the V3 API, or [Price Lists UI](https://support.bigcommerce.com/s/article/Price-Lists).
+
+</div>
+</div>
+</div>
+
+### Product option sets
+
+The V3 Catalog API does not include an endpoint to manage option sets, but it will respect option sets created using the [V2 Option Sets API](https://developer.bigcommerce.com/legacy/v2-catalog-products/v2-option-set-options) or the control panel. Currently, the control panel's Add/Edit Product section consumes the V2 API, and any products created and managed through the control panel will be converted to the V2 model using option sets. If you apply an option set to a V3 product, you will remove the product's variants.
```

Comment:
```suggestion
The V3 Catalog API does not include an endpoint to manage option sets, but it will respect option sets created using the [V2 Option Sets API](https://developer.bigcommerce.com/legacy/v2-catalog-products/v2-option-set-options) or the control panel. On all new stores, the Catalog management control panel interfaces use the V3 API. On some older stores, the control panel's Add/Edit Product section consumes the V2 API, and any products created and managed through the control panel will be converted to the V2 model using option sets. If you apply an option set to a V3 product, you will remove the product's variants.
```

Reply:
+ @becomevocal @bigcommerce/catalog-dt I think we can massage this a bit more.

Reply:
On all new stores, the **Catalog management** control panel interfaces use the V3 API. 

Should this say Products instead of Catalog management to match what is the left pane?

Reply:
Great point. We should use a consistent name with the merchant KB. 

In https://support.bigcommerce.com/s/article/Adding-Products-v3 I see it referred to as:

> the new v3 Add/Edit Product experience

> our built-in Add Product page

> Products › Add.

Any of these could work I think.

Reply:
I've updated this section.

Reply:
@bookernath Can I publish this document or do you want to wait for feedback from @becomevocal and @bigcommerce/catalog-dt ?